src/engine/tfoper.jl

```diff
@@ -2074,9 +2074,9 @@ function tfoper!(gee, M0, T0, p0, a0, Tref, pref,
             cp_al, R_al = gas_pratd(lambdap, nair,
                   pt5, Tt5, ht5, st5, cpt5, Rt5, pcn, 1.0)
 
-            if (ht5 < h5)
-                  println("? ht5 h5 = ", ht5, h5)
-            end
+            # if (ht5 < h5)
+            #       println("? ht5 h5 = ", ht5, h5)
+            # end
```

Comment:
Should we replace this with better error handling? Or have the recent changes made this check unnecessary?